os/net/mac/tsch/tsch-packet.c

```diff
@@ -459,4 +459,18 @@ tsch_packet_parse_eb(const uint8_t *buf, int buf_size,
   return curr_len;
 }
 /*---------------------------------------------------------------------------*/
+/* Set frame pending bit in a packet (whose header was already build) */
+void
+tsch_packet_set_frame_pending(uint8_t *buf, int buf_size)
+{
+  buf[0] |= (1 << 4);
```

Comment:
Might be better to use a `#define` here instead of the constant `4`

Reply:
agree